toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -311,27 +311,10 @@ func PackageNamesFromSingleSystemConfig(systemConfig configuration.SystemConfig)
 // SelectKernelPackage selects the kernel to use for the current installation
 // based on the KernelOptions field of the system configuration.
 func SelectKernelPackage(systemConfig configuration.SystemConfig, isLiveInstall bool) (kernelPkg string, err error) {
-	const (
-		defaultOption = "default"
-		hypervOption  = "hyperv"
-	)
+	const defaultOption = "default"
 
 	optionToUse := defaultOption
 
-	// Only consider Hyper-V for an ISO
-	if isLiveInstall {
-		// Only check if running on Hyper V if there's a kernel option for it
-		_, found := systemConfig.KernelOptions[hypervOption]
-		if found {
-			isHyperV, err := isRunningInHyperV()
```

Comment:
Can we also remove `isRunningInHyperV`? I don't think anything else calls into it.

Reply:
Agreed. Done :)